packages/icons/src/CheckIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import { shadeBlack, shadeWhite } from '@littlespoon/theme/lib/colors/token'
+
+import { CheckIconBase } from './CheckBase'
+
+export interface CheckIconProps extends React.SVGAttributes<SVGElement> {
+  /**
+   * The fill of the component. Defaults to white.
+   */
+  fill?: string
+
+  /**
+   * The stroke of the component. Defaults to black.
+   */
+  stroke?: string
+
+  /**
+   * The size of the component. Defaults to "small".
+   */
+  size?: 'xsmall' | 'small' | 'medium' | 'large'
+}
+
+export default function CheckIcon({
+  fill = shadeWhite,
+  stroke = shadeBlack,
+  size = 'small',
+  ...other
+}: CheckIconProps) {
+  return (
+    <CheckIconBase
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      size={size}
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...other}
```

Comment:
Do we want to support the override of the previous props? If that's not the case, it'll be better to move the spreading before the explicit props.

Reply:
I think we want to override other props